tests/testthat/test-validate_model_metadata_schema.R

```diff
@@ -6,7 +6,7 @@ test_that("Missing files returns an invalid config with an immediate message", {
       "File does not exist"
     )
   })
-  expect_false(out)
+  expect_false(unclass(out))
```

Comment:
NOTE: this is due to [a change in `testthat`](https://hachyderm.io/@zkamvar/113635605411352733) (see https://github.com/r-lib/testthat/pull/2011) where this started failing because this has the `error` class and the method dispatch for an `error` class from `encodeString()` expected a `conditionMessage()` method.

Reply:
Aha! This is good to know as it's been causing problems in other PRs too. Could you add a note about that in the test file?

Reply:
done in b78bc89668bc19b1d3c1c27dce45f682f5c5c4ff